src/general/nqueens.rs

```diff
@@ -0,0 +1,156 @@
+#[allow(unused_imports)]
+use std::env::args;
+
+#[allow(dead_code)]
+#[cfg(not(test))]
+fn main() {
+    let mut board_width = 0;
+
+    for arg in args() {
+        board_width = match arg.parse() {
+            Ok(x) => x,
+            _ => 0,
+        };
+
+        if board_width != 0 {
+            break;
+        }
+    }
+
+    if board_width == 0 || board_width < 2 {
```

Comment:
Please think this condition over

Reply:
I'm sure I did it with good reason. I'll review it.

Reply:
It checks for 0 as apart of its command line parameter checking. The less than 2 was more-or-less a hacking suggestion on what to tinker with for parameter numbers on what board sizes work.

Reply:
I amended that line in the latest commit.